### ISSUE TITLE
I'm on late at all

### ISSUE THREAD

**Issue by francescobianco** [bpkg/bpkg.github.io#79]
Hi @samlikins ,

I'm to you to say Thanks! For your hard work on maintain BPKG website. During this period, my life has been turned upside down by numerous commitments. 

I would like to bring two things to your attention that I hope you will enjoy:

1. The MUSH project   (some similar to Cargo (Rust) but build and for BASH) -  https://github.com/javanile/mush - https://mush.javanile.org/
 
2. A remote_theme  designed to build Package-Manger website like bpkg.github.io

The benefit of a remote theme is that package-mageer-website host only the registry files and not jekyll codebase

Read more: https://talk.jekyllrb.com/t/github-pages-use-of-remote-theme/1737

Added on this I worked on this project  https://github.com/javanile/package-manager

A remote theme for packager manager website (example here: https://www.javanile.org/package-manager/)

- Demo [bpkg.javanile.org](https://bpkg.javanile.org/)
- Codebase https://github.com/javanile/bpkg.javanile.org

cc @jwerle @Potherca

**Comment by jwerle** [comment]
very cool @francescobianco !

**Comment by samlikins** [comment]
So javanil had cooked BPKG... Hmmm!

**Comment by francescobianco** [comment]
Yes @samlikins but not to have an hard fork or a new path, but it is only a convenient way to make experiment with this codebase. I love BPKG, I do not want reinvent the wheel ;-)
